source/_posts/2020-03-18-release-107.markdown

```diff
@@ -139,7 +139,6 @@ Screen recording of the new card picker and stack editor.
 ## New Integrations
 
 - Add platform Ezviz ([@BaQs] - [#30378]) ([ezviz docs]) (new-integration)
-- Add Avri waste collection sensor ([@timvancann] - [#31134]) ([avri docs]) (new-integration)
```

Comment:
These lines should not have been removed, as they are still true. Only the link to the documentation should have been removed.

(this applies to all other occurrences as well).